admin/system/login-proses.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+    include("connection.php");
+    session_start();
+
+    $username = $_POST["username"];
+    $password = $_POST["password"];
```

Comment:
Codacy found an issue: [Detected usage of a possibly undefined superglobal array index: $_POST["password"]. Use isset() or empty() to check the index exists before using it](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5762293)